packages/react/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@puzzlehq/sdk",
+  "displayName": "Puzzle React SDK",
+  "version": "0.3.0",
+  "description": "Your portal to privacy",
+  "main": "./dist/puzzle.cjs.js",
+  "module": "./dist/puzzle.es.js",
+  "browser": "./dist/puzzle.umd.js",
+  "types": "./dist/types/index.d.ts",
+  "exports": {
+    ".": {
+      "import": "./dist/puzzle.es.js",
+      "require": "./dist/puzzle.cjs.js",
+      "browser": "./dist/puzzle.umd.js",
+      "types": "./dist/types/index.d.ts"
+    }
+  },
+  "type": "module",
+  "scripts": {
+    "fetch-fix": "find dist -type f \\( -name '*.js' -o -name '*.cjs' \\) -exec sed -i '' 's/self.fetch[[:space:]]*||/fetch ||/g' {} \\;",
+    "ws-fix": "find ./dist -type f -name 'index*' -exec sed -i '' -e 's/require(\\\"ws\\\")/(() => {try { return require(\\\"ws\\\") } catch (e) { } })()/g' {} +;",
```

Comment:
are these two `-fix` scripts still needed?

Reply:
yeah they're needed, otherwise there will be compilation errors due to walletconnect deps / NextJS bs